front_end/html/footer.html

```diff
@@ -1,21 +1,10 @@
-            <footer class="footer" style="position: fixed; bottom: 0; width: 100%; height: 3.5rem;">
-                <div class="columns">
-                    <div class="column is-one-quarter">
-                        <p>CodeBuddy</p>
-                    </div>
-                    <div class="column is-one-quarter">
-                        <p><a href="/static/about.html">About</a></p>
-                    </div>
-                    <div class="column is-one-quarter">
-                        <p><a href="/static/contact_us.html">Contact Us</a></p>
-                    </div>
-                    <div class="column is-one-quarter">
-                        <p>
-                        </p>
-                    </div>
-                </div>
-            </footer>
-        </div>
+        <footer class="footer">
+            <div class="footer-flex">
+                <div>CodeBuddy</div>
```

Comment:
Will you please dynamically build footer.html and specify the version number of CodeBuddy similar to what is done on [this site](https://www.cbioportal.org)? In build.sh, you'll probably want to put a placeholder in footer.html like {VERSION} and then use the [sed](https://linuxize.com/post/how-to-use-sed-to-find-and-replace-string-in-files/) command to replace that with the contents of the VERSION file.

Reply:
I added a line for this, but wasn't able to test it. Let me know if it works for you.